packages/effects-core/src/vfx-item.ts

```
@@ -3,36 +3,27 @@ import { Quaternion } from '@galacean/effects-math/es/core/quaternion';
   BoundingBoxData, CameraController, HitTestBoxParams, HitTestCustomParams, HitTestSphereParams,
   HitTestTriangleParams, InteractComponent, SpriteComponent,
 } from './plugins';
 import { ParticleSystem } from './plugins';
+import type { TransformProps } from './transform';
 import { Transform } from './transform';
 import type { Constructor, Disposable } from './utils';
 import { removeItem } from './utils';
-import type { EventEmitterListener, EventEmitterOptions, ItemEvent } from './events';
-import { EventEmitter } from './events';
 
 export type VFXItemContent = ParticleSystem | SpriteComponent | CameraController | InteractComponent | undefined | {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)